### PR TITLE
add vector mask menu item

### PR DESCRIPTION
**Pull request by pineapplespatula**
Adding menu item so anyone can ever find this feature.

### CODE REVIEW

File: src/js/actions/layers.js [hunk 1]
```diff
@@ -2161,17 +2161,13 @@ define(function (require, exports) {
      */
     var editVectorMask = function () {
         var toolStore = this.flux.store("tool"),
-            superselectVector = toolStore.getToolByID("superselectVector");
+            curentVectorMaskMode = toolStore.getVectorMode();
```
Comment (mcilroyc) [inline src/js/actions/layers.js]: `curent` :musical_keyboard:

File: src/js/models/menubar.js [hunk 2]
```diff
@@ -235,7 +235,17 @@ define(function (require, exports, module) {
...
                 (document !== null) && hasPreviousHistoryState,
             "later-history":
-                (document !== null) && hasNextHistoryState
+                (document !== null) && hasNextHistoryState,
+            "one-layer-selected-can-have-vector-mask":
+                (document !== null) &&
+                !document.unsupported &&
+                (document.layers !== null) &&
+                (document.layers.selectedNormalized.size === 1) &&
+                (document.layers.selected.every(function (layer) {
```
Comment (mcilroyc) [inline src/js/models/menubar.js]: consider moving this logic into a layerstructure property at some point in the future?

**Comment by mcilroyc** [comment]
this doesn't work for me?  Anyway, back to you to fix the misspelling at least.

**Comment by iwehrman** [comment]
I'm not actually sure if we want to add this to the menu item. @placegraphichere and I are thinking that we want a toolbar icon for this (see #2766), and it's confusing to have it in the menu given that it isn't actually a menu command.

**Comment by pineapplespatula** [comment]
I marked this as UI and review until @placegraphichere chimes in

**Comment by iwehrman** [comment]
I'm closing this out because the only reason I think we otherwise would like this menu item is to make it available in search. But, I think the better thing to do is to just add all (most?) global shortcuts to the search. This will get us things like mask mode as well tool activation, layers panel navigation, etc.

**Pull request closed without merging**